emu.c

```diff
@@ -280,10 +280,10 @@ int main(int parc, char ** pars)
     int ticked = 1;
 
     memset(&emu, 0, sizeof(emu));
-    emu.mCodeMemSize = 65536;
-    emu.mCodeMem     = calloc(emu.mCodeMemSize, sizeof(unsigned char));
-    emu.mExtDataSize = 65536;
-    emu.mExtData     = calloc(emu.mExtDataSize, sizeof(unsigned char));
+    emu.mCodeMemMaxIdx = 65536-1;
```

Comment:
since we're including stdint.h,  you can also use the UINT16_MAX constants instead of 65536, though perhaps that's a seperate PR.

Reply:
Indeed, as to me it feels like a half backed idea. 

I mean, we go via all the burden of making everything configurable, but don't offer any command line option to actually configure it. 😉 

And yes, I'm aiming to add some CLI options.